### ISSUE TITLE
Quod AI - Big Data Engineer - Remote/Singapore - FT or PT

### ISSUE THREAD

**Issue by mikhfilippov** [awesome-jobs/vietnam#1466]
## Location

* Saigon, Vietnam
* Remote

## Salary Expectation

* Competitive

## Requirements
* Java 
* Python
* Minimum 2 years of experience with focus on backend development

### Must Have

* Relevant experience with AWS or Google Cloud.
* Superior analytical, conceptual and problem-solving skills.
* The ability to learn and iterate quickly.
* An obsession with agile and lean principles (GitHub, Trello).

### Ideally

* Experience with complex text parsing and web scraping a plus.
* Experience with data pipelines or ETL a plus.

### Responsibilities

* Work closely with our software architect and data scientist to develop and maintain our core product.
* Design and implement our data pipelines (Java or Python, Elasticsearch, Redis, Apache Beam).
* Work on the implementation of new features, leaving things properly implemented, well documented and delivered on time.
* Always be on the lookout for opportunities to create and improve. From our development process, up to final user’s experiences.

### You’ll be responsible for

* Building the first AI platform that explains source code in plain English.

### Why it would be awesome to work with us

Quod AI is a young and exciting AI/NLP start-up with an unmatched team of engineers and scientists. We’re applying Deep Learning and NLP on source code to empower software engineering teams to develop software products faster and better. We want to revolutionize the way software products are built and to accelerate innovation worldwide. We are funded and backed by SGInnovate and Entrepreneur First.

### Contact
Homepage: https://www.quod.ai/
Quod AI: hello_at_quod_dot_ai (mobile number optional)

**Comment by qcam** [comment]
:robot: Closing this job post because it has been expired :alarm_clock:.
